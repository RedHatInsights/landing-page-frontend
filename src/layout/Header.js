import React from 'react';
import { PageHeader, PageHeaderTitle } from '@red-hat-insights/insights-frontend-components/components/PageHeader';

const Header = () => (
    <PageHeader>
        <PageHeaderTitle title='Manage, automate, and optimize your IT'/>
    </PageHeader>
);

export default Header;
