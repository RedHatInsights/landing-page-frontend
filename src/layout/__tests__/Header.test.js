import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from '../Header';

describe('Header component', () => {
    it('should render correctly', () => {
        const wrapper = shallow(<Header />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
