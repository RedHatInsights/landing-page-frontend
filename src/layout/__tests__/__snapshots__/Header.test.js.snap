// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render correctly 1`] = `
<PageHeader>
  <PageHeaderTitle
    title="Manage, automate, and optimize your IT"
  />
</PageHeader>
`;
